### DISCUSSION TITLE
Some details about deepxde in heat equation

### DISCUSSION THREAD

**Discussion by w-venton** [lululxvi/deepxde#1222] [Q&A]
Hi, lululxvi. I am confused about the unit of pde. Like: the area defined in some example is x,y in (0,1). I can't find that whether the unit of it is meter or any other. If I want to define a area whose unit is μm, how can I do? Also, I want to know the unit of other valuation. Can you tell me the details or some useful webpage? I have seen many issues but I haven't found it. Thank you very much!!

**Comment by lululxvi** [comment]
This has nothing to do with DeepXDE. Check a PDE book.
